From a89e5ec5eb6e120cf69e6725a99c32e65bac6072 Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@apache.org>
Date: Thu, 3 Jan 2013 20:55:09 +0000
Subject: [PATCH 0760/1357] HDFS-4302. Fix fatal exception when starting NameNode with DEBUG logs. Contributed by Eugene Koontz.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1428591 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit ce03fd2e43ba5104de54a8e3420713fd31aa2ae3)
---
 .../hdfs/server/namenode/FSEditLogLoader.java      |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSEditLogLoader.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSEditLogLoader.java
index e97953d..2c06ead 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSEditLogLoader.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSEditLogLoader.java
@@ -126,10 +126,6 @@ public class FSEditLogLoader {
     long numTxns = (lastTxId - expectedStartingTxId) + 1;
     long lastLogTime = now();
 
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("edit log length: " + in.length() + ", start txid: "
-          + expectedStartingTxId + ", last txid: " + lastTxId);
-    }
     try {
       while (true) {
         try {
-- 
1.7.0.4

